README.md

```diff
@@ -41,19 +67,15 @@ In that case, just click `Restart IDE` and confirm that action in a message box.
 Git Machete IntelliJ Plugin is a port of a great console tool &mdash; [git-machete](https://github.com/VirtusLab/git-machete#git-machete), into an IntelliJ plugin.
 Git Machete can help you manage your repository branches and keep it in sync with each other and its counterparts on a remote repository.
 
-<details>
-<summary><strong>Where to find the plugin tab</strong></summary>
-<br/>
+### Where to find the plugin tab
 
 Git Machete IntelliJ Plugin is available under `Git` tool window in the `Git Machete` tab.
 You can also use `Ctrl` + `Alt` + `Shift` + `M` shortcut to open it.
 
 ![](docs/open_git_machete.gif)
-</details>
 
-<details>
-<summary><strong>Discover</strong></summary>
-<br/>
+
```

Comment:
is not a single empty line enough (here and everywhere XD)?

Reply:
I think two blank lines between sections make this document more clear

Reply:
Fine, be consistent then